image-download

```diff
@@ -85,9 +85,7 @@ def find(name, stores, latest, quiet):
     for store in stores:
         url = urllib.parse.urlparse(urllib.parse.urljoin(store, name))
 
-        args = []
-        if ca := get_host_ca(url.netloc):
-            args += ['--cacert', ca]
+        args = get_curl_ca_arg(url.netloc)
```

Comment:
I wonder if we should make this part of `s3.sign_curl()` to better match the fact that `s3.urlopen()` takes care of this for itself.

This would have the unfortunate side-effect of meaning that we need to "do it ourselves" when we don't have an S3 key.  Which brings the next question: perhaps, instead of failing, calling `s3.urlopen()` or `s3.sign_curl()` should simply do unsigned requests.

That's a follow-up, and I'm happy to take care of it.